model/manifest_test.go

```diff
@@ -64,6 +64,7 @@ func TestFindManifest(t *testing.T) {
 func TestManifestUnmarshal(t *testing.T) {
 	expected := Manifest{
 		Id:               "theid",
+		IconPath:          "https://example.org/icon.svg",
```

Comment:
nit: not that it matters, but since this field is path, should the example also be a relative path?

Reply:
Good catch, fixed :+1: